tests/test_struct.py

```diff
@@ -49,6 +49,12 @@ def testInit(self):
         for name, val in self.valDict.items():
             self.assertEqual(getattr(s, name), val)
 
+    def testInitException(self):
+        """Test Struct.__init__
```

Comment:
I suggest a more informative description, something along the lines of "test that names cannot start with __" or "test Struct constructor errors"

Reply:
Oops.. yes I forgot to update that description!